asciidoc/courses/genai-mcp-build-custom-tools-python/modules/1-getting-started/lessons/2-setup/lesson.adoc

```diff
@@ -0,0 +1,151 @@
+= Setting Up Your Development Environment
+:type: lesson
+:order: 2
+:lab: {repository-link}
+:language: Python
+:disable-cache: true
+:branch: main
+
+
+We have created a GitHub Codespace for you to experiment with building MCP servers without any local setup required.
+
+The Codespace comes pre-configured with Python, the MCP SDK, and all the tools you need. You'll just need to configure your Neo4j database credentials to get started.
+
+In this lesson, you'll learn how to launch your Codespace and set up your environment variables.
+
+
+include::../../../../../../shared/courses/codespace/get-started.adoc[]
+
+
+[%collapsible]
+.Develop on your local machine
+====
+You will need link:https://python.org[Python 3.10 or higher^] installed.
+
+This course uses link:https://docs.astral.sh/uv/[uv^] as the Python package manager.
+Install `uv` by following the link:https://docs.astral.sh/uv/getting-started/installation/[installation instructions^].
+
+Clone the link:{repository-link}[github.com/neo4j-graphacademy/genai-mcp-build-custom-tools-python] repository:
+
+[source,bash]
+----
+git clone https://github.com/neo4j-graphacademy/genai-mcp-build-custom-tools-python
+----
+
+Navigate to the project directory:
+
+[source,bash]
+----
+cd genai-mcp-build-custom-tools-python
+----
+====
```

Comment:
the devcontainer includes instructions to install node. Is this required for the local installation?